.appveyor.yml

```diff
@@ -0,0 +1,88 @@
+version: 1.0.{build}

```

Comment:
do want/need something better here, resembling actual to branches, maybe?

Reply:
I don't know -- if there's something that other projects do, we can do it too?  The current version setting is not great, but it seems "good enough for v1".